### ISSUE TITLE
Add Zoom in and Zoom out buttons for Poi map

### ISSUE THREAD

**Issue by bahaaTuffaha** [digitalfabrik/integreat-app#3478]
### Describe the Problem

This zoom in and zoom out buttons will allow keyboard users to be able to zoom just by focusing on the these buttons.

### Describe Your Preferred Solution

- Add 2 buttons on the right of the screen on top of the map one for zooming in the map camera and one for zooming out. (Design needs discussion)
- Make sure the buttons are responsive when the bottom-sheet minimize/maximize.

### Describe Possible Alternatives

N/A.

### Additional Information
Check the keyboard Focus implementation  #3369

**Comment by bahaaTuffaha** [comment]
This low prio (after the implementation of the new design system)

**Comment by lunars97** [comment]
Really good idea 👍

**Comment by steffenkleinle** [comment]
Is there no other way to do that? How does it work on e.g. google maps, afaics there are also no zoom in/out buttons. Perhaps we can also add buttons that are not visible on the screen but focusable with the keyboard, similar to https://github.com/digitalfabrik/integreat-app/pull/3645. What do you think @lunars97 @bahaaTuffaha. Also concerns web mobile.

**Comment by bahaaTuffaha** [comment]
> Is there no other way to do that? How does it work on e.g. google maps, afaics there are also no zoom in/out buttons. Perhaps we can also add buttons that are not visible on the screen but focusable with the keyboard, similar to [#3645](https://github.com/digitalfabrik/integreat-app/pull/3645). What do you think [@lunars97](https://github.com/lunars97) [@bahaaTuffaha](https://github.com/bahaaTuffaha). Also concerns web mobile.

google maps has zoom buttons but we could try the hidden focusable ones first

<img width="244" height="174" alt="Image" src="https://github.com/user-attachments/assets/dfbf72d7-e5d3-45d4-af33-b68bd7e62405" />

**Comment by steffenkleinle** [comment]
> > Is there no other way to do that? How does it work on e.g. google maps, afaics there are also no zoom in/out buttons. Perhaps we can also add buttons that are not visible on the screen but focusable with the keyboard, similar to [#3645](https://github.com/digitalfabrik/integreat-app/pull/3645). What do you think [@lunars97](https://github.com/lunars97) [@bahaaTuffaha](https://github.com/bahaaTuffaha). Also concerns web mobile.
> 
> google maps has zoom buttons but we could try the hidden focusable ones first

Not for me, at least not in the native app and not on web mobile. Only on web desktop (where we also have zoom buttons).

**Comment by lunars97** [comment]
> > > Is there no other way to do that? How does it work on e.g. google maps, afaics there are also no zoom in/out buttons. Perhaps we can also add buttons that are not visible on the screen but focusable with the keyboard, similar to [#3645](https://github.com/digitalfabrik/integreat-app/pull/3645). What do you think [@lunars97](https://github.com/lunars97) [@bahaaTuffaha](https://github.com/bahaaTuffaha). Also concerns web mobile.
> > 
> > 
> > google maps has zoom buttons but we could try the hidden focusable ones first
> 
> Not for me, at least not in the native app and not on web mobile. Only on web desktop.

yes, it is only on web desktop

**Comment by lunars97** [comment]
i just checked keyboard accessibility on the map on native, seems like it is possible to move around with key arrows, but not possible  to zoom in and out. And key arrows and zooming out and in with + key are not possible on google maps...

**Comment by steffenkleinle** [comment]
Okay, in that case I'd say we should only display the zoom buttons on web desktop (as we currently do) and add hidden but with keyboard focusable buttons to zoom in and out on both native and web mobile.
Alternatively, we could think of using some shortcut (e.g. CTRL + `+`/`-`). What do you think? 
Anyway, its not an ui-ux ticket then.

**Comment by bahaaTuffaha** [comment]
> Okay, in that case I'd say we should only display the zoom buttons on web desktop (as we currently do) and add hidden but with keyboard focusable buttons to zoom in and out on both native and web mobile. Alternatively, we could think of using some shortcut (e.g. CTRL + `+`/`-`). What do you think? Anyway, its not an ui-ux ticket then.

Sounds good. I think the shortcuts only on web desktop

**Comment by steffenkleinle** [comment]
> > Okay, in that case I'd say we should only display the zoom buttons on web desktop (as we currently do) and add hidden but with keyboard focusable buttons to zoom in and out on both native and web mobile. Alternatively, we could think of using some shortcut (e.g. CTRL + `+`/`-`). What do you think? Anyway, its not an ui-ux ticket then.
> 
> Sounds good. I think the shortcuts only on web desktop

Then lets not do any shortcuts, not worth it.
